lib/config/presets.js

```diff
@@ -0,0 +1,159 @@
+  pinVersions: {
+    description:
+      'Use version pinning (maintain a single version only and not semver ranges)',
+    pinVersions: true,
+  },
+  preserveSemverRanges: {
+    description:
+      'Preserve (but continue to upgrade) any existing semver ranges',
+    pinVersions: false,
+  },
+  pinOnlyDevDependencies: {
+    description:
+      'Pin dependency versions for devDependencies and preserve ranges for others',
+    dependencies: { presets: ['preserveSemverRanges'] },
+    devDependencies: { presets: ['pinVersions'] },
+    optionalDependencies: { presets: ['preserveSemverRanges'] },
+    peerDependencies: { presets: ['preserveSemverRanges'] },
+  },
+  separateMajorReleases: {
+    description:
+      'Separate different major releases of a dependencies into separate PRs',
+    separateMajorReleases: true,
+  },
+  separatePatchReleases: {
+    description:
+      'Separate patch and minor releases of dependencies into separate PRs',
+    separatePatchReleases: true,
+  },
+  combinePatchMinorReleases: {
+    description:
+      'Combine any patch and minor upgrades of a dependency into the same PR',
+    separatePatchReleases: false,
+  },
+  renovatePrefix: {
+    description: 'Use "renovate/" as prefix for all branch names',
+    branchprefix: 'renovate/',
+  },
+  semanticPrefixChore: {
+    description:
+      'Use "chore(deps):" as semantic prefix for commit messages and PR titles',
+    semanticPrefix: 'chore(deps):',
+  },
+  semanticPrefixFix: {
+    description:
+      'Use "fix(deps):" as semantic prefix for commit messages and PR titles',
+    semanticPrefix: 'fix(deps):',
+  },
+  disablePeerDependencies: {
+    description: 'Do not renovate peerDependencies versions/ranges',
+    peerDependencies: { enabled: false },
+  },
+  semanticPrefixFixDepsChoreOthers: {
+    description:
+      'If semantic commits detected, use "fix(deps):" for dependencies and "chore(deps):" for all others',
+    dependencies: { presets: ['semanticPrefixFix'] },
+    devDependencies: { presets: ['semanticPrefixChore'] },
+    optionalDependencies: { presets: ['semanticPrefixChore'] },
+    peerDependencies: { presets: ['semanticPrefixChore'] },
+  },
+  unpublishSafe: {
+    description:
+      'Set a status check to warn when upgrades < 24 hours old can be unpublished',
+    unpublishSafe: true,
+  },
+  prImmediately: {
+    description: 'Raise PRs immediately (after branch is created)',
+    prCreation: 'immediate',
+  },
+  prNotPending: {
+    description:
+      'Raise PRs only once branch status is no longer in "pending" state',
+    prCreation: 'not-pending',
+  },
+  automergeDisabled: {
+    description: 'Do not automerge any upgrades',
+    automerge: 'none',
+  },
+  automergePatch: {
+    description: 'Automerge patch upgrades if they pass tests',
+    automerge: 'patch',
+  },
+  automergeMinor: {
+    description: 'Automerge patch or minor upgrades if they pass tests',
+    automerge: 'minor',
+  },
+  automergeMajor: {
```

Comment:
heya! :)

1. there's a typo: you wrote `it` instead of `if`
2. shouldn't `automerge` be `any` here? Or you are thinking for something new?